src/Silk.Data/Models/Tag.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Silk.Data.Models
+{
+    public class Tag
+    {
+        public int Id { get; set; }
+        public int Uses { get; set; }
+        
+        public string Name { get; set; }
+        public string Content { get; set; }
+        
+        public DateTime CreatedAt { get; set; }
+        
+        public ulong OwnerId { get; set; }
+        public ulong GuildId { get; set; }
+        
+        public int? OriginalTagId { get; set; }
+        public Tag? OriginalTag { get; set; }
+
+        public List<Tag>? Aliases { get; set; }
```

Comment:
So why isn't aliases just a list of strings?  It seems like you'll have to do extra work to make sure tag content is the same across all aliases and you don't do alias of alias, etc.

Reply:
Moderation reasons; aliases are almost the same as tags, and they're actually fairly easy to update as I can just check for all tags that have a specific FK. It also allows for servers to see who's making excessive aliases, and restricts who can delete aliases, as aliases have owners.